### DISCUSSION TITLE
docs: What is exactly "EntityDTO" ?

### DISCUSSION THREAD

**Discussion by alcalyn** [mikro-orm/mikro-orm#6294] [General]
I was wondering about the difference between `ArrayCollection.getItems` and `ArrayCollection.toArray`, one returns `EntityDTO` but I didn't found what is it one the documentation.

I just found this occurence: https://mikro-orm.io/docs/defining-entities#default-values

and this discussion: https://github.com/mikro-orm/mikro-orm/discussions/5186

**Comment by B4nan** [comment]
The name should be quite clear, its a DTO for the entity. Entities have runtime-specific mapping like `Reference` or `Collection` wrappers, while a DTO is a simple POJO, so those wrappers are processed (e.g. instead of `Collection` you get an array).

> I was wondering about the difference between ArrayCollection.getItems and ArrayCollection.toArray

`getItems()` should be also quite clear, it gives you the items the collection holds - entities. While `toArray` is a serialization method, returning POJOs.

I am not sure if we need yet another section in the docs to describe this, but I would love to have the jsdoc for those (and as such, they will be rendered as part of the API docs).
